src/js/edu-benefits/utils/helpers.jsx

```diff
@@ -142,3 +142,7 @@ export const preferredContactMethodLabels = {
   phone: 'Phone'
 };
 
+export const relationshipLabels = {
+  spouse: 'Spouse',
+  child: 'Child'
```

Comment:
Oh, the 1990e just has 'Child' and 'Spouse' :(
I'll check with Alex to see if 'Child' should be more expansive, but 1990e definitely won't be using 'Surviving Spouse' so maybe this set of labels shouldn't be shared after all?